### ISSUE TITLE
Request Failed Status 400

### ISSUE THREAD

**Issue by kevinjgibb** [samrum/homebridge-onstar#286]
Plugin was working fine for several months but now I receive the error while trying to execute commands. I have updated the UUID, restarted bridge, reinstalled plugin, all to no avail.

**Comment by mentaluproar** [comment]
I’m seeing this issue too. I suspect GM changed something again and we will need to wait for another update.  GMs shenanigans keep screwing with external services like this.

**Comment by billchilders** [comment]
Seeing this as well.

**Comment by Z1mDMan** [comment]
Also having this issue

**Comment by FishNinjas** [comment]
Glad I’m not the only one, I’ve been troubleshooting this but giving up for now. This is the only way I’ve been able to schedule remote start for my Chevy so I hope this is a fixable issue!

**Comment by pmcavey** [comment]
I am seeing this issue also. Tried changing UUID but that didn’t change anything.

**Comment by pmcavey** [comment]
Just to rule out a problem with my Synology, I erased and re-setup a spare Synology installed homebridge and added only the Onstar plugin and get the same error.

**Comment by donniehanna** [comment]
I am having the same problem also, seems to have started last week, I have rolled back to a previous version but that does not seem to resolve it, is there a plan to resolve this soon?

**Comment by deboy69** [comment]
@samrum

**Comment by pcaruso2** [comment]
Same issue here. Both of my GM cars return 400 error.

**Comment by jpenns** [comment]
My chevy app is working fine, but I'm also having the same issue with the plug-in

**Comment by chicagospeed** [comment]
So issue seems to be MFA? 

https://github.com/BigThunderSR/homeassistant-addons-onstar2mqtt/issues/744

**Comment by Z1mDMan** [comment]
> So issue seems to be MFA? 
> 
> https://github.com/BigThunderSR/homeassistant-addons-onstar2mqtt/issues/744

@samrum can the homebridge plugin get a bump to the new OnStarJS please? Assuming that will maybe fix this issue.

**Comment by joker3287** [comment]
Still getting the Bad Request error 400. I have tried to install the update to included TOTP for the multifactor authentication but I can’t seem to get it to work. I am really new to this and probably doing it wrong. But since it is not part of the actual plug in I am lost. Please update to include

**Comment by Z1mDMan** [comment]
> Still getting the Bad Request error 400. I have tried to install the update to included TOTP for the multifactor authentication but I can’t seem to get it to work. I am really new to this and probably doing it wrong. But since it is not part of the actual plug in I am lost. Please update to include

Check out homebridge-OnStar-dev which works fine now.

**Comment by joker3287** [comment]
@Z1mDMan How do I install that into Homebridge?

**Comment by Z1mDMan** [comment]
> @Z1mDMan How do I install that into Homebridge?

Search for homebridge-onstar-dev in the plugins section.
https://github.com/reartvapps/homebridge-onstar-dev#readme

**Comment by usmcfiredog** [comment]
@samrum  have you abandoned this project?

**Comment by samrum** [comment]
Yes, I'll be archiving this repo/plugin. It's dependent on onstarjs which is similarly no longer maintained and will also be archived.

Similar to the way onstarjs has been replaced by onstarjs2, I'd recommend using newer community alternatives.
